package.json

```diff
@@ -31,19 +31,25 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@aws-sdk/client-s3": "^3.540.0",
+    "@aws-sdk/client-secrets-manager": "^3.540.0",
+    "@aws-sdk/lib-storage": "^3.540.0",
+    "@aws-sdk/types": "^3.535.0",
     "@dvsa/cvs-microservice-common": "^0.9.6",
-    "aws-sdk": "^2.1383.0",
+    "@smithy/util-utf8": "^2.3.0",
+    "aws-sdk-client-mock": "^4.0.0",
     "aws-xray-sdk": "^3.3.4",
     "exceljs": "^4.3.0",
     "js-yaml": "^3.13.1",
     "moment": "^2.29.4",
     "moment-timezone": "^0.5.27",
     "node-yaml": "^4.0.1",
     "request": "^2.88.2",
-    "request-promise": "^4.2.5"
+    "request-promise": "^4.2.6",
+    "serverless-dependency-invoke": "file:./packages/serverless-dependency-invoke"
```

Comment:
request-promise is probably not being used in v3 because of the await async syntax
serverless-dependency-invoke can go in devDependencies